roachtest: hibernate failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20141989?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20141989?buildTab=artifacts#/hibernate) on release-25.1 @ [cc91ad3e666ff6b167c85d9e139b167d330db6c8](https://github.com/cockroachdb/cockroach/commits/cc91ad3e666ff6b167c85d9e139b167d330db6c8):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.9-dev-cc91ad3e666ff6b167c85d9e139b167d330db6c8
Tests run against hibernate 6.6.20
13148 Total Tests Run
13147 tests passed
1 test failed
2495 tests skipped
2 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: org.hibernate.orm.test.hql.BulkManipulationTest.testInsertWithManyToOne - unknown (unexpected)
For a full summary look at the hibernate artifacts 
An updated blocklist (hibernateBlockList) is available in the artifacts' hibernate log
test artifacts and logs in: /artifacts/hibernate/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20141989/hibernate/1752127465638/1752139823178)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-52382

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20174638?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20174638?buildTab=artifacts#/hibernate) on release-25.1 @ [acbd9ddbd3fc6a10321c7ca1b4b822aa03f23ac5](https://github.com/cockroachdb/cockroach/commits/acbd9ddbd3fc6a10321c7ca1b4b822aa03f23ac5):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.9-dev-acbd9ddbd3fc6a10321c7ca1b4b822aa03f23ac5
Tests run against hibernate 6.6.20
13148 Total Tests Run
13147 tests passed
1 test failed
2495 tests skipped
2 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: org.hibernate.orm.test.hql.BulkManipulationTest.testInsertWithManyToOne - unknown (unexpected)
For a full summary look at the hibernate artifacts 
An updated blocklist (hibernateBlockList) is available in the artifacts' hibernate log
test artifacts and logs in: /artifacts/hibernate/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20174638/hibernate/1752558421226/1752570364551)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #149885 roachtest: hibernate failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3]
- #149884 roachtest: hibernate failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.1.9-rc]
- #149883 roachtest: hibernate failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.3.16-rc]
- #149881 roachtest: hibernate failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2.3-rc]
- #149880 roachtest: hibernate failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2]
- #149879 roachtest: hibernate failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-master]
- #149878 roachtest: hibernate failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.3]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/hibernate_blocklist.go
+++ b/pkg/cmd/roachtest/tests/hibernate_blocklist.go
@@ -21,6 +21,7 @@ var hibernateSpatialIgnoreList = blocklist{
 var hibernateIgnoreList = blocklist{
 	"org.hibernate.orm.test.batch.BatchTest.testBatchInsertUpdate":                                       "flaky",
 	"org.hibernate.orm.test.bulkid.OracleInlineMutationStrategyIdTest.testDeleteFromEngineer":            "flaky",
+	"org.hibernate.orm.test.hql.BulkManipulationTest.testInsertWithManyToOne":                            "https://hibernate.atlassian.net/browse/HHH-19332",
 	"org.hibernate.serialization.SessionFactorySerializationTest.testUnNamedSessionFactorySerialization": "flaky",
 	"org.hibernate.serialization.SessionFactorySerializationTest.testNamedSessionFactorySerialization":   "flaky",
 	"org.hibernate.test.batch.BatchTest.testBatchInsertUpdate":                                           "flaky",

